### ISSUE TITLE
FR: An easier way to upload/manage minutes on the website.

### ISSUE THREAD

**Issue by jubjamie** [backstagetechnicalservices/website#67]
It is currently extremely tedious to download and rename a pdf version of the google doc of some minutes and then re-upload them, title it again, rag it etc. Especially when you have to do about 30 every semester. Is there an easier way to do this using [Make Docs Public - Google Drive](https://support.google.com/docs/answer/183965?co=GENIE.Platform%3DDesktop&hl=en) or is it net easier to just churn through every document, download, rename, re-upload etc.?

As it's already quite clunky to search for minutes is it worth looking at being able to attach a file directly in the event so you can find the meeting in the calendar where the name and date are already there and then you can just upload or link to the google docs minutes there with less faffing of renaming the docs and tagging etc.? Ideally an upload here would still upload it to resources but with title inferred and the tags auto-selected so they appear there too? Maybe make it a paperwork category only visible in the meeting event category like the event report?

I guess on solution is to keep the website up to date every week and avoid doing it bulk but...

**Comment by bnjns** [comment]
It's always been the plan to properly integrate Google Docs into the resources section, but the challenge is the access control so it'll take time. In the meantime, if you keep on top of it, it isn't that much effort.

You can already link resources to events. You don't yet see them when viewing the event, but that's because the paperwork view is currently being revamped to do this.

**Comment by ACascarino** [comment]
I really disagree with this being closed so quickly as I think there's a lot of scope for conversation around how we manage resources and their links to events and clients that this feature request opens up - and besides, "it's always been the plan" is great and all but is there a visible place that these development points are logged and/or viewable? How, for example, are we supposed to know that the paperwork view is being revamped to display resources without some kind of entity (issue report, feature request, development plan) that tells us this?

**Comment by bnjns** [comment]
I closed the issue because I answered both sides in my reply, and keeping it open is just a duplication and unnecessary.

Releasing the resources section without fully integrating Google Docs was an informal committee discussion a couple of years ago, but properly finishing it off has always been on my to-do list. There's a very set plan for it, so not much to discuss around it at all - the only reason it doesn't yet exist is that there's always something more urgent that needs doing.

As this to-do list is not, and never will be, made public there's nothing wrong with having brought this up as a feature request as of course, you won't know about the plan. But equally, if an issue is unnecessary I am allowed to close it with a brief explanation.

I also want to point out that yet again this issue describes 2 features/issues. This makes it really difficult to actually discuss the issue correctly and marking it as complete is a nightmare because a commit will never address both issues.
